### PR TITLE
BD hadron track monitoring in standard validation sequence 91X (master)

### PR DESCRIPTION
**Pull request by smoortga**
same PR as https://github.com/cms-sw/cmssw/pull/18011 but in the master branch (91X)

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: Validation/Configuration/python/globalValidation_cff.py [hunk 1]
```diff
@@ -91,6 +92,7 @@
                                  + pfTauRunDQMValidation
                                  + bTagPlotsMCbcl
                                  + L1Validator
+				 + bdHadronTrackValidationSeq
```
Comment (dmitrijus) [inline Validation/Configuration/python/globalValidation_cff.py]: do not introduce tab  characters in python files, please.

File: Validation/RecoB/python/BDHadronTrackValidation_cff.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    addTagInfos = cms.bool(True)
+)
+
+# my analyzer
+from Validation.RecoB.BDHadronTrackMonitoring_cfi import *
+from SimTracker.TrackerHitAssociation.tpClusterProducer_cfi import *
+BDHadronTrackMonitoringAnalyze.PatJetSource = cms.InputTag('patJetsBDHadron')
+
+bdHadronTrackValidationSeq = cms.Sequence(patJetCorrections
+					* patJetCharge*patJetPartonMatch
```
Comment (dmitrijus) [inline Validation/RecoB/python/BDHadronTrackValidation_cff.py]: tabs again.

**Review by dmitrijus — CHANGES_REQUESTED** [review]

**Comment by smoortga** [comment]
@dmitrijus , fixed

**Comment by dmitrijus** [comment]
thanks

**Comment by dmitrijus** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by smoortga** [comment]
Hi all,

Thanks for putting effort in reviewing this PR with me. Which approval-flags do we still need set to get this PR merged? (and who is responsible for that?)

Cheers,
Seth

**Comment by perrotta** [comment]
+1
Validation stuff
Code looks reasonable

**Comment by perrotta** [comment]
hold
(while checking whether, and to what extent, the producers run in the newly added sequence add  up to the saved output)

**Comment by perrotta** [comment]
unhold
(explicitely verified that the new products are never saved with the usual reco-related event contents)

**Comment by davidlange6** [comment]
+1

**Pull request merged**